### PR TITLE
Complete Build Script for JS

### PR DESCRIPTION
**Pull request by lpzchrstn**
# Finished Build Script for JS
This PR contains these additional features:
- A package.json file containing the dependencies and the scripts for bundling js:
  - Build Script
  - Prod Script
  - Watch Script
  - Rollup Configuration Script
  - Rollup Configuration Script for prod

## Notes:
- The build, prod and watch script can be viewed in #1
- All js files should be imported in `src/js/scripts.js`

### CODE REVIEW

**Review by joshuacerbito — CHANGES_REQUESTED** [review]
Please review my comments from the [other PR](https://github.com/lpzchrstn/sass-lpzchrstn/pull/1) as those are also applicable to this PR.

Also, we don't include compiled assets to the repo (e.g. the `dist` files) since that cause a lot of conflicts especially when working with other developers. Please untrack the `dist` files, and then update `.gitignore` and ignore everything inside the `dist` folder.

**Comment by lpzchrstn** [comment]
> Please review my comments from the [other PR](https://github.com/lpzchrstn/sass-lpzchrstn/pull/1) as those are also applicable to this PR.
> 
> Also, we don't include compiled assets to the repo (e.g. the `dist` files) since that cause a lot of conflicts especially when working with other developers. Please untrack the `dist` files, and then update `.gitignore` and ignore everything inside the `dist` folder.

Done adding changes for this.

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+node_modules/
+dist/
```
Comment (joshuacerbito) [inline .gitignore:2]: Please always add an empty line at the end of every file

**Review by joshuacerbito — CHANGES_REQUESTED** [review]

**Review by joshuacerbito — CHANGES_REQUESTED** [review]
Everything looks good except for the Merge Conflicts.
Please fix the conflicts first, then this PR's good for merging.

**Comment by lpzchrstn** [comment]
> Everything looks good except for the Merge Conflicts. Please fix the conflicts first, then this PR's good for merging.

Fixed the Merge Conflict.

**Pull request merged**